scripts/exregional_run_post.sh

```diff
@@ -93,8 +93,8 @@ to the temporary work directory (DATA_FHR):
   DATA_FHR = \"${DATA_FHR}\"
 ===================================================================="
 else
-  if [ ${FCST_MODEL} = "fv3gfs_aqm" ]; then
-    post_config_fp="${PARMdir}/upp/postxconfig-NT-fv3lam_cmaq.txt"
+  if [ "${CPL_AQM}" = "TRUE" ]; then
+    post_config_fp="${HOMEdir}/sorc/AQM-utils/parm/postxconfig-NT-fv3lam_cmaq.txt"
```

Comment:
The AQM param files should probably goto `${HOMEdir}/param` -- easiest would be a symlink like `upp-aqm` under `$HOMEdir/parm`

Reply:
modified.